examples/GAN/gan_mnist_pytorch/README.md

```diff
@@ -0,0 +1,16 @@
+
```

Comment:
Add a title as a `#` line, and remove the emphasis for the first sentence ("This folder ...")?